learn/core_concepts/indexes.md

```diff
@@ -130,3 +131,11 @@ Typo tolerance is a built-in feature that helps you find relevant results even w
 You can update the typo tolerance settings using the [update settings endpoint](/reference/api/settings.md#update-settings) or the [update typo tolerance endpoint](/reference/api/typo_tolerance.md#update-typo-tolerance).
 
 [Learn more about typo tolerance](/learn/configuration/typo_tolerance.md)
+
+### Faceting
+
+Facets are a specific use-case of filters in Meilisearch, whether something is a facet or filter depends on your UI and UX design. Like filters, you need to add your facets to [`filterableAttributes`](/reference/api/filterable_attributes.md#update-filterable-attributes) then, make a search query using the [`filter` search parameter](/reference/api/search.md#filter).
+
+By default, Meilisearch returns `100` facet values for each faceted field. You can change this using the [update settings endpoint](/reference/api/settings.md#update-settings) or the [update faceting settings endpoint](/reference/api/faceting.md#update-faceting-settings).
+
+[Learn more about faceting](/learn/advanced/filtering_and_faceted_search.md)
```

Comment:
I think we should change this sentence and the others like it in this page. They feel very rough, which I'm unsure whether it's because we don't offer much explanation of what type of content the user will see or, and this is extremely banal, because we're missing a period.

In any case, out of scope for this PR.

Reply:
Agreed, but we use this format everywhere when we direct users to specific guides. We should discuss this and update accordingly